src/types/nitro.ts

```diff
@@ -31,11 +31,19 @@ export type NitroDynamicConfig = Pick<
   "runtimeConfig" | "routeRules"
 >;
 
+export interface NitroRuntimeConfigApp {
+  baseURL: string;
+  [key: string]: unknown;
```

Comment:
Untyped keys were any since before 2.5. Changing any to unknown (at least for me) is a breaking change. When users where depending on any type it will basically break their type checks even if safer or better. Can we override it for nuxt?

Reply:
Seems Nuxt itself is still any 

<img width="1044" alt="image" src="https://github.com/unjs/nitro/assets/5158436/def980f4-3930-4933-bab4-ac486512af8c">

<img width="496" alt="image" src="https://github.com/unjs/nitro/assets/5158436/89b26f45-f875-4439-aae4-891e17d08e5d">

Reply:
I have made change to make PR mergeable. Would be happy to discuss about next steps plan of adopting unknown...